### PR TITLE
[planet-finance] Add Strategy

### PR DESCRIPTION
**Pull request by ash-greninja-poke**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Comment by ash-greninja-poke** [comment]
How much time it will take to review can i get an estimate?

**Comment by ChaituVR** [comment]
@ash-greninja-poke Strategy is failing to generate scores, please try running strategy tests in your local, Thanks
```
cannot estimate gas; transaction may fail or may require manual gas limit (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":48,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":
```

**Comment by ash-greninja-poke** [comment]
> @ash-greninja-poke Strategy is failing to generate scores, please try running strategy tests in your local, Thanks
> 
> ```
> cannot estimate gas; transaction may fail or may require manual gas limit (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":48,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":
> ```

how to add the gas limit manually?

**Comment by ChaituVR** [comment]
@ash-greninja-poke Not sure.. looks like something is wrong with the contract call.. maybe try to refactor your code and see where it is going wrong

also instead of copying the entire ABI into the strategy  list only ABI functions that are used in the strategy

**Comment by ash-greninja-poke** [comment]
I have now tested it properly can you please review it now?

**Comment by ChaituVR** [comment]
LGTM @bonustrack

**Comment by ash-greninja-poke** [comment]
@ChaituVR  how much time it will take to review?

**Comment by ChaituVR** [comment]
> @ChaituVR how much time it will take to review?

@ash-greninja-poke It depends, Maybe less than one week. @bonustrack will do the review soon

File: src/strategies/index.ts [hunk 1]
```diff
@@ -207,5 +208,6 @@ export default {
   'balancer-delegation': balancerDelegation,
   'infinityprotocol-liquidity-pools': infinityProtocolPools,
   'aave-governance-power': aaveGovernancePower,
-  cake
+  cake,
+  planetFinance
```
Comment (bonustrack) [inline src/strategies/index.ts:212]: The strategy name should use kebab case, i recommend to change to this:
```suggestion
  'planet-finance': planetFinance
```

Comment (bonustrack) [reply src/strategies/index.ts:212]: @ash-greninja-poke The strategy folder too. If you can change both the strategy will be ready to merge.

**Comment by ash-greninja-poke** [comment]
@ChaituVR can you review it now?

File: src/strategies/index.ts (hunk 1 above)
Comment (ash-greninja-poke) [reply src/strategies/index.ts]: @bonustrack i have made the changes can you please review it now?

**Comment by ash-greninja-poke** [comment]
@ChaituVR what is the issue now?

**Comment by ash-greninja-poke** [comment]
@ChaituVR everything seems to be good can we please merge it now?

**Comment by ChaituVR** [comment]
> @ChaituVR everything seems to be good can we please merge it now?

@ash-greninja-poke 
Looks like sometimes it is taking more than 20 seconds to fetch votes for 500 addresses, looks like you have so many contract calls in code, is it possible to make it reduce the number of calls to make it more scalable? (while having more voters)

**Comment by ash-greninja-poke** [comment]
I tried but i have to make that contract calls so it is not possible to scale it more.

**Comment by ash-greninja-poke** [comment]
@ChaituVR @bonustrack can you guys merge it?

**Comment by ash-greninja-poke** [comment]
I have scaled it more .

**Comment by ash-greninja-poke** [comment]
@ChaituVR @bonustrack

**Comment by ash-greninja-poke** [comment]
@bonustrack @ChaituVR can we please merge it now?

**Review by bonustrack — APPROVED** [review]
Looks good

**Pull request merged**